L1Trigger/L1TCommon/plugins/L1TConfigDumper.cc

```diff
@@ -162,168 +162,194 @@
+  edm::ESGetToken<L1CaloEtScale, L1HtMissScaleRcd> EToken_;
+  edm::ESGetToken<L1CaloEtScale, L1HfRingEtScaleRcd> FToken_;
+  edm::ESGetToken<L1MuTriggerScales, L1MuTriggerScalesRcd> GToken_;
+  edm::ESGetToken<L1MuTriggerPtScale, L1MuTriggerPtScaleRcd> HToken_;
+  edm::ESGetToken<L1MuGMTScales, L1MuGMTScalesRcd> IToken_;
+  edm::ESGetToken<L1MuCSCTFConfiguration, L1MuCSCTFConfigurationRcd> JToken_;
+  edm::ESGetToken<L1MuCSCTFAlignment, L1MuCSCTFAlignmentRcd> KToken_;
+  edm::ESGetToken<L1MuCSCPtLut, L1MuCSCPtLutRcd> LToken_;
+  edm::ESGetToken<L1MuDTEtaPatternLut, L1MuDTEtaPatternLutRcd> MToken_;
+  edm::ESGetToken<L1MuDTExtLut, L1MuDTExtLutRcd> NToken_;
+  edm::ESGetToken<L1MuDTPhiLut, L1MuDTPhiLutRcd> OToken_;
+  edm::ESGetToken<L1MuDTPtaLut, L1MuDTPtaLutRcd> PToken_;
+  edm::ESGetToken<L1MuDTQualPatternLut, L1MuDTQualPatternLutRcd> QToken_;
+  edm::ESGetToken<L1MuDTTFParameters, L1MuDTTFParametersRcd> RToken_;
+  edm::ESGetToken<L1RPCConfig, L1RPCConfigRcd> SToken_;
+  edm::ESGetToken<L1RPCConeDefinition, L1RPCConeDefinitionRcd> TToken_;
+  edm::ESGetToken<L1RPCHsbConfig, L1RPCHsbConfigRcd> UToken_;
+  edm::ESGetToken<L1RPCBxOrConfig, L1RPCBxOrConfigRcd> VToken_;
+  edm::ESGetToken<L1MuGMTParameters, L1MuGMTParametersRcd> WToken_;
+  edm::ESGetToken<L1RCTParameters, L1RCTParametersRcd> XToken_;
+  edm::ESGetToken<L1CaloEcalScale, L1CaloEcalScaleRcd> YToken_;
+  edm::ESGetToken<L1CaloHcalScale, L1CaloHcalScaleRcd> ZToken_;
+  edm::ESGetToken<L1GctJetFinderParams, L1GctJetFinderParamsRcd> AAToken_;
+  edm::ESGetToken<L1GtBoardMaps, L1GtBoardMapsRcd> BBToken_;
+  edm::ESGetToken<L1GtParameters, L1GtParametersRcd> CCToken_;
+  edm::ESGetToken<L1GtStableParameters, L1GtStableParametersRcd> DDToken_;
+  edm::ESGetToken<L1GtTriggerMask, L1GtTriggerMaskVetoAlgoTrigRcd> EEToken_;
+  edm::ESGetToken<L1GtTriggerMenu, L1GtTriggerMenuRcd> FFToken_;
+  edm::ESGetToken<L1GtPsbSetup, L1GtPsbSetupRcd> GGToken_;
+  edm::ESGetToken<L1CaloGeometry, L1CaloGeometryRecord> HHToken_;
+  edm::ESGetToken<L1MuDTTFMasks, L1MuDTTFMasksRcd> IIToken_;
+  edm::ESGetToken<L1MuGMTChannelMask, L1MuGMTChannelMaskRcd> JJToken_;
+  edm::ESGetToken<L1RCTChannelMask, L1RCTChannelMaskRcd> KKToken_;
+  edm::ESGetToken<L1RCTNoisyChannelMask, L1RCTNoisyChannelMaskRcd> LLToken_;
+  edm::ESGetToken<L1GctChannelMask, L1GctChannelMaskRcd> MMToken_;
+  edm::ESGetToken<L1GtPrescaleFactors, L1GtPrescaleFactorsAlgoTrigRcd> NNToken_;
+  edm::ESGetToken<L1GtPrescaleFactors, L1GtPrescaleFactorsTechTrigRcd> OOToken_;
+  edm::ESGetToken<L1GtTriggerMask, L1GtTriggerMaskAlgoTrigRcd> PPToken_;
+  edm::ESGetToken<L1GtTriggerMask, L1GtTriggerMaskTechTrigRcd> QQToken_;
+  edm::ESGetToken<L1GtTriggerMask, L1GtTriggerMaskVetoTechTrigRcd> RRToken_;
 };
 
 L1TConfigDumper::L1TConfigDumper(const edm::ParameterSet& iConfig)
-
-{}
-
-L1TConfigDumper::~L1TConfigDumper() {}
+    : AToken_(esConsumes()),
+      BToken_(esConsumes()),
+      CToken_(esConsumes()),
+      EToken_(esConsumes()),
```

Comment:
```suggestion
      CToken_(esConsumes()),
      DToken_(esConsumes()),
      EToken_(esConsumes()),
```
Maybe one call was missed? :)

Reply:
Good catch! Guess I don't know the alphabet :P

Reply:
fixed